### ISSUE TITLE
app: Add functionality to change background in chat diorama too

### ISSUE THREAD

**Issue by alisaad673** [webaverse/app#2556]
Right now, we can change background for character tab diorama, this seems to be a cool feature. We should also add this functionality to chat diorama.

https://user-images.githubusercontent.com/35926530/157896275-52cf5e5d-1e81-40c1-b3ee-aed963bf103e.mp4

**Comment by avaer** [comment]
Good idea! The plan is to make it unique and configurable for each character.
